setup.cfg

```diff
@@ -1,13 +1,83 @@
 [metadata]
 description-file = README.rst
+url = https://github.com/physiopy/peakdet
+download_url = https://github.com/physiopy/peakdet
+author = peakdet developers
+maintainer = Ross Markello, Stefano Moia
+maintainer_email = rossmarkello@gmail.com, s.moia@bcbl.eu
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+license = Apache-2.0
+description = A python toolbox for reproducible physiological data analysis
```

Comment:
I think you can do better at this description

Reply:
What about "A Python toolbox for the identification and processing of peaks and troughs in physiological data"?